tutorials/phase_contrast_calculation.ipynb

```diff
@@ -0,0 +1,216 @@
+      "| 22 |        608.605 |        0.319 |      15.000 |            31.797 |            9.939 |           61.172 |              14.264 |\n",
+      "| 23 |        573.206 |        0.319 |      15.000 |            29.947 |           10.769 |           60.195 |              13.851 |\n",
+      "| 24 |        568.470 |        0.319 |      15.000 |            29.700 |           14.383 |           48.230 |              10.060 |\n",
+      "| 25 |        511.694 |        0.319 |      15.000 |            26.733 |           12.772 |           45.690 |               9.921 |\n",
+      "| 26 |        447.939 |        0.319 |      15.000 |            23.403 |            9.744 |           47.009 |              11.023 |\n",
+      "| 27 |        425.067 |        0.319 |      15.000 |            22.208 |            7.985 |           45.592 |              10.404 |\n",
+      "| 28 |        444.729 |        0.340 |      16.000 |            21.783 |            6.764 |           43.053 |               9.768 |\n",
+      "| 29 |        411.481 |        0.319 |      15.000 |            21.498 |            6.422 |           40.659 |               9.709 |\n",
+      "| 30 |        376.518 |        0.319 |      15.000 |            19.671 |            2.320 |           36.068 |               9.612 |\n",
+      "| 31 |        346.105 |        0.319 |      15.000 |            18.082 |            0.708 |           31.087 |               8.194 |\n",
+      "| 32 |        398.673 |        0.340 |      16.000 |            19.527 |            6.960 |           25.958 |               5.768 |\n",
+      "| 33 |        404.968 |        0.298 |      14.000 |            22.669 |            6.520 |           27.082 |               5.350 |\n",
+      "| 34 |        459.282 |        0.319 |      15.000 |            23.995 |            3.834 |           33.333 |               8.443 |\n",
+      "| 35 |        498.825 |        0.298 |      14.000 |            27.923 |           19.414 |           35.482 |               4.921 |\n",
+      "| 36 |        478.383 |        0.298 |      14.000 |            26.778 |           18.437 |           33.333 |               4.887 |\n",
+      "| 37 |        428.900 |        0.255 |      12.000 |            28.010 |           18.974 |           34.603 |               4.824 |\n",
+      "| 38 |        432.452 |        0.255 |      12.000 |            28.242 |           21.758 |           34.554 |               4.228 |\n",
+      "| 39 |        416.934 |        0.255 |      12.000 |            27.228 |           23.419 |           32.357 |               3.139 |\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Print the stats_table\n",
+    "pc_obj.print_stats_table()\n",
+    "# Save the table with the results into a .csv file\n",
+    "csv_path = os.path.join(OUTPUT_DIR, \"pc_left_output.csv\")\n",
+    "pc_obj.save_stats_table_to_csv(csv_path)"
```

Comment:
I'd just call this `to_csv` or `stats_to_csv` (and maybe rename `print_stats_table()` just to `print_stats()`). Really long variable names make code difficult to read and aren't really adding any extra information. The printed table looks much better though!